ocs_ci/ocs/ui/base_ui.py

```diff
@@ -11,6 +13,7 @@
 import zipfile
 from functools import reduce
 
+import pandas
```

Comment:
Why we need 2 import `import pandas` and ` import pandas as pd`

Reply:
accidentally, will fix